packages/swapper/src/api.ts

```diff
@@ -76,4 +85,9 @@ export interface Swapper {
    * Get max swap balance (minus fees) for sell asset
    */
   getSendMaxAmount(args: SendMaxAmountInput): Promise<string>
+
+  /**
+   * Get a boolean if swapper supports an asset
+   */
+  isSupportedAssets(args: SupportedAssetInput): boolean
```

Comment:
What does `supported` mean?

We talked about the possibility of an asset being available on only one side of a trade, like being a "sell" or "buy" asset but not both.